MAUI/TextInputLayout/Supported-input-views.md

```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
 {% endhighlight %}
 {% endtabs %}
 
-![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
\ No newline at end of file
+![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
+
+## Picker
+
+To initialize the `Picker` control and launch it in each platform, refer to the `getting started with picker` documentation.
+
+{% tabs %}
+{% highlight XAML %}
+
+<inputLayout:SfTextInputLayout Hint="Fruit" 
+                               HelperText="Select a fruit"
+                               ContainerType="Outlined" >
+   <picker:Picker>
+        <Picker.ItemsSource>
+            <x:Array Type="{x:Type x:String}">
+                <x:String>Apple</x:String>
+                <x:String>Orange</x:String>
+                <x:String>Strawberry</x:String>
+            </x:Array>
+        </Picker.ItemsSource>
+   </picker:Picker>
+</inputLayout:SfTextInputLayout> 
+
+{% endhighlight %}
+{% highlight C# %}
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Fruit"; 
+inputLayout.HelperText = "Select a fruit";
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new SfPicker(); 
+
+{% endhighlight %}
+{% endtabs %}
+
+![Picker](images/SupportedInputViews/Picker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms Picker` as input view of the text input layout.
+
+## TimePicker
+
+To initialize the `TimePicker` control and launch it in each platform, refer to the `getting started with time picker` documentation.
+
+{% tabs %}
+{% highlight XAML %}
+
+ <inputLayout:SfTextInputLayout Hint="Time" 
+                               HelperText="Select a start time"
+                               ContainerType="Outlined" >
+    <timepicker:TimePicker Time="12:00"/>
+ </inputLayout:SfTextInputLayout>
+
+{% endhighlight %}
+{% highlight C# %}
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Time"; 
+inputLayout.HelperText = "Select a start time";
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new TimePicker() {Time="12:00"}; 
+
+{% endhighlight %}
+{% endtabs %}
+
+![TimePicker](images/SupportedInputViews/TimePicker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms TimePicker` as input view of the text input layout.
+
+## DatePicker
```

Comment:
Add time date picker documentation link: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/DatePicker

Reply:
Addressed the changes.